scripts/release.sh

```diff
@@ -139,17 +144,19 @@ package="$(grep *windows_amd64.zip manifest | cut -f2 -d' ')"
 cat -<<EOF
       {
         "platform": "Windows Binaries (64-bit)",
+        "ref": "win",
         "sha256":"$(sha256sum $package | cut -f1 -d' ')",
         "code":[
           "wget https://dl.influxdata.com/telegraf/releases/$package -UseBasicParsing -OutFile $package",
-          "Expand-Archive .\\$package -DestinationPath 'C:\\Program Files\\InfluxData\\telegraf\\'"
+          "Expand-Archive .\\\\$package -DestinationPath 'C:\\\Program Files\\\InfluxData\\\telegraf\\\'"
```

Comment:
I think we need four backslashes every time. Some of these only have three
```suggestion
          "Expand-Archive .\\\\$package -DestinationPath 'C:\\\\Program Files\\\\InfluxData\\\\telegraf\\\\'"
```